diff --git a/jit/Makefile.am b/jit/Makefile.am
index 93b884e..5b01ae6 100644
--- a/jit/Makefile.am
+++ b/jit/Makefile.am
@@ -2,6 +2,7 @@ BUILT_SOURCES = jit-opcode.c jit-interp-opcode.h jit-interp-opcode.c
 
 lib_LTLIBRARIES = libjit.la
 
+pkgconfigdir = ${libdir}/pkgconfig
 pkgconfig_DATA = libjit.pc
 
 libjit_la_SOURCES = \
diff --git a/jitdynamic/Makefile.am b/jitdynamic/Makefile.am
index 4465d9c..f2fa390 100644
--- a/jitdynamic/Makefile.am
+++ b/jitdynamic/Makefile.am
@@ -1,6 +1,7 @@
 
 lib_LTLIBRARIES = libjitdynamic.la
 
+pkgconfigdir = ${libdir}/pkgconfig
 pkgconfig_DATA = libjitdynamic.pc
 
 libjitdynamic_la_SOURCES = \
diff --git a/jitplus/Makefile.am b/jitplus/Makefile.am
index b9e78b7..d403282 100644
--- a/jitplus/Makefile.am
+++ b/jitplus/Makefile.am
@@ -1,6 +1,7 @@
 
 lib_LTLIBRARIES = libjitplus.la
 
+pkgconfigdir = ${libdir}/pkgconfig
 pkgconfig_DATA = libjitplus.pc
 
 libjitplus_la_SOURCES = \
